cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -46,7 +46,8 @@ type MasterConfiguration struct {
 	// SelfHosted enables an alpha deployment type where the apiserver, scheduler, and
 	// controller manager are managed by Kubernetes itself. This option is likely to
 	// become the default in the future.
-	SelfHosted bool
+	SelfHosted     bool
+	SkipTokenPrint bool
```

Comment:
I still don't think this should be in the API.
It should be Init struct that's for CLI params

This is a CLI-specific thing, not something that should be a part of an API